marketplace_standard_app_api/routers/transformation.py

```diff
@@ -109,9 +111,7 @@ async def update_transformation(
         404: {"description": "Not found."},
     },
 )
-async def get_transformation_state(
-    transformation_id: TransformationId,
-) -> TransformationStateResponse:
+async def get_transformation_state() -> TransformationStateResponse:
```

Comment:
How are you supposed to get the state if you do not provide an id for the transformation?

Reply:
This is a mistake. I was not intended to remove it. I will add it again.